tests/src/test/scala/spire/laws/LawTests.scala

```diff
@@ -33,21 +33,21 @@ class LawTests extends FunSuite with Discipline {
   }
 
   // Float and Double fail these tests
-  checkAll("Int",        RingLaws[Int].euclideanRing)
-  checkAll("Long",       RingLaws[Long].euclideanRing)
+  checkAll("Int",        RingLaws[Int].cRing)
+  checkAll("Long",       RingLaws[Long].cRing)
```

Comment:
Two things (that should be documented):

- signedGCDRing is an add-on on top of other tests (so that I don't need to define signedEuclideanRing, signedField, orderedGCDRing, orderedEuclideanRing, ...)

- the limited range types cannot be tested with Signed, due to wrap on overflow, see the next PR #634 for a solution (let us discuss it there)